### PR TITLE
Fix some option notations

### PR DESCRIPTION
**Pull request by chromato99**
Hi.
While reading the document, there were some parts where the notation was ambiguous, so I would like to suggest corrections. Please check it out and leave a comment.
The 8.0 branch also has the same problem.

### CODE REVIEW

**Comment by chromato99** [comment]
There was also an error in the streaming backup document, so I modified it further.
If you run it according to the current document, an unknown argument error will occur as shown below, so I added the `--target-dir` option to the command.
```bash
vm-1:~$ sudo xtrabackup --backup --stream=xbstream ./ | nc dbp-test-vm-3.kep.k9d.in 9999
2024-02-29T13:57:55.127916+09:00 0 [Note] [MY-011825] [Xtrabackup] recognized server arguments: --datadir=/mysql/data --server-id=1 --open_files_limit=8161 --innodb_adaptive_hash_index=0 --innodb_max_dirty_pages_pct=90 --innodb_io_capacity=500 --innodb_buffer_pool_size=4294967296 --innodb_flush_log_at_trx_commit=1
2024-02-29T13:57:55.128106+09:00 0 [Note] [MY-011825] [Xtrabackup] recognized client arguments: --backup=1 --stream=xbstream
2024-02-29T13:57:55.128126+09:00 0 [ERROR] [MY-011825] [Xtrabackup] unknown argument: './'
```

**Review by patrickbirch — APPROVED** [review]
fine

File: docs/take-streaming-backup.md [hunk 1]
```diff
@@ -59,13 +59,13 @@ were both compressed and encrypted, they must be decrypted before they are uncom
 |---------|------|
 | Stream the backup into an archived named `backup.xbstream` | `$ xtrabackup --backup --stream=xbstream --target-dir=./ > backup.xbstream`|
 | Stream the backup into a compressed archive named `backup.xbstream`| `$ xtrabackup --backup --stream=xbstream --compress --target-dir=./ > backup.xbstream` |
-| Encrypt the backup | `$ xtrabackup --backup --stream=xbstream --target-dir=./ > backup.xbstream gzip -`` | openssl des3 -salt -k “password” backup.xbstream.gz.des3` |
+| Encrypt the backup | `$ xtrabackup --backup --stream=xbstream > backup.xbstream gzip -`` | openssl des3 -salt -k “password” backup.xbstream.gz.des3` |
```
Comment (satya-bodapati) [inline docs/take-streaming-backup.md:62]: This command line no longer works. Do you plan to fix this as a separate PR?

The command that worked for me (using my system config):
`/xtrabackup --user=root --socket=/home/satya/WORK/PS-8.0-trunk/bld-rel/mysql-test/var/tmp/mysqld.1.sock  --backup --stream=xbstream  |gzip  | openssl des3 -salt -k 'password' -out  backup.xbstream.gz.des3`

File: docs/take-streaming-backup.md [hunk 2]
```diff
@@ -59,13 +59,13 @@ were both compressed and encrypted, they must be decrypted before they are uncom
 |---------|------|
 | Stream the backup into an archived named `backup.xbstream` | `$ xtrabackup --backup --stream=xbstream --target-dir=./ > backup.xbstream`|
```
Comment (satya-bodapati) [inline docs/take-streaming-backup.md:60]: Another suggestion is to remove the $ in all the command line strings. Its easy to copy this command line string. One less work to remove the $ sign

**Review by satya-bodapati — CHANGES_REQUESTED** [review]

**Review by alina-derkach-oaza — APPROVED** [review]

File: docs/take-streaming-backup.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/take-streaming-backup.md:62]: Sure, I'll create a separate PR and fix docs according to your comments.

**Review by patrickbirch — APPROVED** [review]
approved

**Review by alina-derkach-oaza — APPROVED** [review]

**Pull request merged**